yarn.lock

```diff
@@ -2459,6 +2459,23 @@
   resolved "https://registry.yarnpkg.com/@metamask/safe-event-emitter/-/safe-event-emitter-2.0.0.tgz#af577b477c683fad17c619a78208cede06f9605c"
   integrity sha512-/kSXhY692qiV1MXu6EeOZvg5nECLclxNXcKCxJ3cXQgYuRymRHpdx/t7JXfsK+JLjwA1e1c1/SBrlQYpusC29Q==
 
+"@mm-snap/controllers@^0.0.6":
```

Comment:
not sure why this got updated when i ran `yarn setup`

Reply:
I'm pretty sure it's because there's a mismatch between versions among the monorepo packages and it's pulling that one from the registry. My general take on this for the time being is: meh.